_includes/manuals/2.3/1.2_release_notes.md

```diff
@@ -3,6 +3,13 @@
 This section will be updated prior to the next release.
 
 ### Headline features
+**Host Registration**
+New simplified process of registering hosts to the Foreman. With utilization of provisioning templates,
+already running hosts can be registered to the Foreman with one command.
+For more details about registration process:
+* New page _All Hosts > Register Host_
+* Provisioning templates _Global Registration_ & _Linux registration default_
+* [RFC - Simple & automatic host registration WF](https://community.theforeman.org/t/rfc-simple-automatic-host-registration-wf/19588)
```

Comment:
We should probably also add a manual section about this, but doesn't have to be right now (as long as we get it in by the time 2.3 is GA)